core/src/main/kotlin/in/specmatic/stub/api.kt

```diff
@@ -70,6 +71,13 @@ fun createStubFromContracts(
     host: String = "localhost",
     port: Int = 9000
 ): ContractStub {
+    val validContractPaths = contractPaths.filter { path ->
+        File(path).extension in CONTRACT_EXTENSIONS
+    }
+    if (validContractPaths.isEmpty()) {
+        throw ContractException("No valid specification file found at given paths : $contractPaths")
```

Comment:
contractPaths here is a list. Can you confirm that it appears as a CSV when the error message is printed?

The same applies to the change to createStubFromContracts on line 96 in this file.